packages/@aws-cdk/aws-scheduler-alpha/lib/input.ts

```diff
@@ -41,7 +41,7 @@ export abstract class ScheduleTargetInput {
 }
 
 class FieldAwareEventInput extends ScheduleTargetInput {
-  constructor(private readonly input: any) {
+  constructor(private readonly input: any, private readonly toJsonString: boolean) {
```

Comment:
There is probably a better implementation approach but since this is only used internal/private in the class, I will fix the behaviour for now, then improve the design for easier readability and maintainability in subsequent PRs.

Reply:
I think its fine honestly